src/Test.Utilities/Test.Utilities.csproj

```diff
@@ -17,13 +17,6 @@
   <ItemGroup>
     <ProjectReference Include="..\TestReferenceAssembly\TestReferenceAssembly.csproj" />
   </ItemGroup>
-  <ItemGroup>
```

Comment:
Do I need to add the nuget packages for those to avoid the exception I am mentioning?